controllers/socialAuth.js

```diff
@@ -1,22 +1,33 @@
 import dotenv from 'dotenv';
+import crypto from 'crypto';
 import models from '../models';
 import Auth from '../helpers/auth';
 
 dotenv.config();
-
+const { FRONT_END_URL } = process.env;
 const { users } = models;
-
+const algorithm = 'aes-256-ctr';
+const { CRYPTO_PASSWORD } = process.env;
 export default class User {
+  static enc(data) {
```

Comment:
What were you trying to achieve here?

Reply:
atleast securing the return object by encrypting it and sending it to the frontend as a token

Reply:
Okay!